config/config.example.yml

```diff
@@ -64,13 +64,15 @@ gravatar_default: identicon
 
 # Setup your deploy options for capistrano.
 deployment:
-  hosts:
-    web: errbit.example.com
-    app: errbit.example.com
-    db: errbit.example.com
-  repository: https://github.com/errbit/errbit.git
+  servers:
+    errbit.example.com:
+      roles:
+        - app
+        - db
+        - web
+      user: deploy
```

Comment:
`roles` and `user`  will be passed to `server` in production.rb as `opt`.

Having several `server` with same name doesn't seem to work - that's why I changed a way to organize by server, not by role.